### PR TITLE
Fix browserWindows[] indexing

### PR DESCRIPTION
**Pull request by HeilTec**
Using the current index when removing closed window from list allows the application to open, close and re-open windows in any order.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -37,10 +37,10 @@ module.exports = (glob, options) => {
   // Add each created BrowserWindow to list of maintained items
   app.on('browser-window-created', (e, bw) => {
     browserWindows.push(bw)
-    let i = browserWindows.indexOf(bw)
 
     // Remove closed windows from list of maintained items
     bw.on('closed', function () {
+      let i = browserWindows.indexOf(bw) // Must use current index
```
Comment (yan-foto) [inline main.js]: Wow! Does the index change during its life cycle? 😨

**Comment by yan-foto** [comment]
Will merge ASAP. Sorry for the delay and thanks for the PR.

File: main.js (hunk 1 above)
Comment (HeilTec) [reply main.js]: The index of a window in the array changes when the splice function is called during the 'onclosed' event of an earlier window.

Try this:
Open one and two. Close one. Change files. Notice: Two does not reload.
After both are opened:  browserWindows[0] === one && browserWindows[1] === two
Closing one leaves it as: browserWindows[0] two

It only happens if browser windows are dynamically opened and closed by the app.

Comment (yan-foto) [reply main.js]: That makes perfect sense. I should be ashamed of not seeing this! 😅

**Comment by yan-foto** [comment]
Merged and published on [npm](https://www.npmjs.com/package/electron-reload). Sorry for this unnecessary delay and thanks for the PR. 🙇

**Pull request merged**